setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_packages, setup
+
+
+version = '0.0.1'
+
+
+setup(
+    author='WildFoundry Ltd',
+    author_email='info@wildfoundry.com',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)',
```

Comment:
We will be including content from other libraries with differing licenses. See #1.